doc/connecting/topics/as_connecting-to-google-sheets.adoc

```diff
@@ -0,0 +1,54 @@
+// This assembly is included in the following assemblies:
+// Upstream: connecting/master.adoc
+// Downstream: connecting-fuse-online-to-applications-and-services/master.adoc
+
+:context: connectors
+[id='connecting-to-google-sheets_{context}']
+= Connect to Google Sheets
+:context: sheets
```

Comment:
Just want to note here that you are setting the value of the "context" document attribute to "sheets". That's fine, but every xref must end with "_sheets" and there are instances of xrefs that end in "_google-sheets". I'll try to catch them but wanted to note this here in case I miss any.

Reply:
I agree, xrefs should be updated. I will do this. As I have used the Google calendar docs as a reference we might also update these xrefs, too.